config.ini.example

```diff
@@ -59,6 +67,70 @@ mkzero!*@* = all_permissions
 channels = #bytespeicher-testing
 filename = logs/{host}/{channel}-{date:%Y-%m-%d}.log
 
+[plugins.bytehelp]
+location = http://technikkultur-erfurt.de/bytespeicher:anfahrt
+dates = http://technikkultur-erfurt.de/bytespeicher:veranstaltungen
+versorgung = http://technikkultur-erfurt.de/bytespeicher:versorgung
+
 [plugins.autotopic]
-topic = Test topic mit 100% Funktionen
-channels = #bytespeicher-testing
+topic = Testtopic mit 100% Funktionen
+
+[plugins.cccongress]
+url = https://fahrplan.events.ccc.de/congress/YEAR/Fahrplan/schedule.json
+# cache = /tmp/cccongress.cache
+# announce_minutes = 15
+
+[plugins.dates]
+url = http://www.google.com/calendar/ical/2eskb61g20prl65k2qd01uktis%40group.calendar.google.com/public/basic.ics
+# timedelta = 21
+
+[plugins.fuel]
+apikey = your_apikey
+# lat = 50.9827792
+# lng = 11.0394426
+# rad = 10
+
+[plugins.parking]
+# url = parking_url
+
+[plugins.rss]
+# RSS-Name =
+#     URL to RSS feed
+#     Tempfile for caching
+#     feed type (dokuwiki, wordpress, github, redmine)
+Wiki =
+    http://www.technikkultur-erfurt.de/feed.php
+    /tmp/feed_wiki.cache
+    dokuwiki
+Website =
+    https://bytespeicher.org/feed/
+    /tmp/feed_website.cache
+    wordpress
+BytebotCommits =
+    https://github.com/Bytespeicher/Bytebot/commits/master.atom
+    /tmp/feed_bytebot.cache
+    github
+Redmine =
+    http://redmine.bytespeicher.org/issues.atom
+    /tmp/feed_rm_bytespeicher.cache
+    redmine
+SpacestatusCommits =
+    https://github.com/Bytespeicher/space-status/commits/master.atom
+    /tmp/feed_spacestatus.cache
+    github
+
+[plugins.shorturl]
+clarifai_app_id = yourappid
+clarifai_app_secret = yourappsecret
+
+[plugins.status]
```

Comment:
shouldn't that be a more global option, as its not a plugin option per se but rather an option for a library function?